src/TicketingSystem.BusinessLogic/Enums/PaymentState.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace TicketingSystem.BusinessLogic.Enums
+{
+    public enum PaymentState
+    {
```

Comment:
Do you suppose to have different PaymentState in DAL and BLL layers? This is odd. I would recommend creating a Common project and moving all the Enums to reuse in other projects there.

Reply:
No, that was done just to follow sequential dependency of DAL < BLL < PL without extra references. But this makes sense  because having DAL enums and BLL enums leads to extra mapping

Reply:
Enums were moved into a separate Common project to use them without mapping, namespace was updated